src/core/lender/distributor/distributor.sol

```diff
@@ -82,42 +81,41 @@ contract Distributor is DSNote {
     }
 
     function borrow(address tranche, uint currencyAmount) internal returns(uint) {
-        OperatorLike tranche = OperatorLike(tranche);
+        TrancheLike tranche = TrancheLike(tranche);
 
         uint available = tranche.balance();
         if (currencyAmount > available) {
             currencyAmount = available;
         }
 
-        tranche.borrow(address(manager.pile()), currencyAmount);
+        tranche.borrow(shelf, currencyAmount);
         return currencyAmount;
     }
 
     //  method      repayTranches
     //  available   total available currency for repaying the tranches
```

Comment:
maybe fix this to the @xxx notation?

Reply:
yes that's the plan for all contracts anyway